### ISSUE TITLE
A strange distutils stub hangs around on PyPI.org 

### ISSUE THREAD

**Issue by wimglenn** [pypi/support#3660]
PyPI UI shows a project for Distutils with a 1.0 release:

https://pypi.org/project/Distutils/#history
https://pypi.org/project/Distutils/1.0/
https://pypi.org/simple/distutils/
https://pypi.org/pypi/Distutils/1.0/json

There are no files or download links.
Could these be removed, so that these urls all become 404?

Rationale: I've seen confusion from people getting a `ModuleNotFoundError: No module named 'distutils'` on Python 3.12 thinking that they should be installing distutils from PyPI.

**Comment by merwok** [comment]
@jaraco you’re one of the maintainers: do you recall why a pypi project was created?

**Comment by jaraco** [comment]
Yes. I was considering releasing [distutils](/pypa/distutils) as a public package and then vendoring that package into Setuptools (mostly as a means to facilitate the vendoring consistently with other packages, but unlikely as a way for users to download or use that package independently). Thusfar, I've not actually done so, because I'd not felt comfortable with the unwanted consequences (users downloading it and using it independently). I'm okay with deleting the project, just as long as no one else can register it. I realize I'm essentially squatting on it, but only because it's a reserved name already and if it's used by anyone, it should be used be the pypa/distutils project.

That's interesting that there's a 1.0 release without any files. I forget how I did that. I guess I probably just ran something like the 'register' operation. Or maybe I uploaded an artifact and deleted it.

I'm not sure what's the right thing to do here. I'm not even sure what all the options are. But if it's an option to delete it without making it available to others, let's do that.

**Comment by miketheman** [comment]
@jaraco In Feb 2024, warehouse learned about names that exist(ed) in the Python stdlib and denies uploads for them, sourced from https://pypi.org/project/stdlib-list/

Thus, you should be able to log into PyPI, delete the distutils project, and nobody else should be able to upload that name again.
